packages/headless/src/features/analytics/analytics-utils.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {createMockState} from '../../test';
```

Comment:
I'd prefer if we renamed the file to end with `.test.ts` to be consistent with the rest of the headless project.